schema/newschma.yaml

```diff
@@ -0,0 +1,32 @@
+root:
+  parts: # We dropped the SEQUENCE information from the excel sheet
+   name: string
+   role: enum
+   circular: boolean # isn't this implicit from role ?
```

Comment:
Circular indicates which of two defined ontology terms to add (see SBOL spec Component.type)

Reply:
These right ?

```
DNA (Deoxyribonucleic acid) https://identifiers.org/SBO:0000251 12
RNA (Ribonucleic acid) https://identifiers.org/SBO:0000250 13
Protein (Polypeptide chain) https://identifiers.org/SBO:0000252 14
Simple Chemical https://identifiers.org/SBO:0000247 15
Non-covalent complex https://identifiers.org/SBO:0000253 16
Functional Entity
```

Reply:
You want Table 3:
```
linear http://identifiers.org/SO:0000987
circular http://identifiers.org/SO:0000988
```